RELEASE-NOTES.md

```diff
@@ -1,3 +1,32 @@
+### 0.11.0
+#### Compatibility
+- golem-messages: 2.24.0
+- golem-smart-contracts-interface: 1.6.1
+
+#### Endpoints and uses cases in general
+- Bugfix: Change concent response from ServiceRefused to Http404 (#1051).
+- Bugfix: Validate protocol version (#1057).
+- Bugfix: Fix non nesting atomic to verify transaction nesting in runtime instead of on startup (#1053).
+- Bugfix: Wrong jpeg handling in verification use case (#1046).
+
+#### Payments
+- Preventing multiple settlement payments for the same subtasks (#1009, #1039).
+- Bugfix: Get list of payments does not search right range (#1067).
```

Comment:
`Get list of payments` -> `get_list_of_payments` or `The function that finds payments on the blockchain`.

These description should be really more high level. We should mention how this effects the final product rather than what is being changed in the code. In this case something like this would be even more appropriate:
``` markdown
- Bugfix: Concent was not ignoring unconfirmed blocks when searching for payments (#1067).
```

Reply:
Done